pyproject.toml

```
@@ -30,7 +30,7 @@ exclude = ["src/momento/internal/codegen.py"]
 [tool.poetry.dependencies]
 python = ">=3.7,<3.12"
 
-momento-wire-types = "0.31.1"
+momento-wire-types = "^0.39"
```

Let's pin the version

```suggestion
momento-wire-types = "0.39.0"
```

---

Why's that? Wire types should be backwards compatible.

---

That's true they should be. Pinning the version is just a black and white best practice so that what we test in CI is what users install.

---

That's a great point. For the wire types probably. We should also reevaluate how we pinned grpcio as that is likely to be used by popular Google libraries.